upup/pkg/fi/cloudup/terraform/literal.go

```diff
@@ -76,16 +79,39 @@ func sortLiterals(v []*Literal, dedup bool) ([]*Literal, error) {
 		})
 	}
 
+	return proxies, nil
+}
+
+// SortLiterals sorts a list of Literal, by key.  It does so in-place
+func SortLiterals(v []*Literal) {
+	proxies, err := buildSortProxies(v)
+	if err != nil {
+		// Very unexpected
+		glog.Fatalf("error processing terraform Literal: %v", err)
+	}
+
 	sort.Sort(byKey(proxies))
 
-	var sorted []*Literal
+	for i := range proxies {
+		v[i] = proxies[i].literal
+	}
+}
+
+func DedupLiterals(v []*Literal) ([]*Literal, error) {
```

Comment:
Missing code comment. Sorry to nit pick, but I have been asked to cleanup my docs many times.

Reply:
Fixed